android/src/main/java/io/mosip/tuvali/exception/handlers/VerifierExceptionHandler.kt

```diff
@@ -9,10 +9,24 @@ import io.mosip.tuvali.verifier.exception.VerifierException
 class VerifierExceptionHandler(val sendError: (String, ErrorCode) -> Unit) {
   private val logTag = Util.getLogTag(javaClass.simpleName)
 
-  fun handleException(e: VerifierException){
+  fun handleException(e: VerifierException) {
     val rootCause = ExceptionUtils.getRootBLECause(e)
+    val crcFailureCount = ExceptionUtils.getCRCFailureCount(e)
+    val totalChunkCount = ExceptionUtils.getTotalChunkCount(e)
 
     Log.e(logTag, "Handling Verifier Exception: ", e)
-    sendError(e.message ?: "Something went wrong in Verifier: $rootCause", rootCause.errorCode)
+
+    val crcErrorLogString = if (crcFailureCount != null)
+      "CRCFailureCount:$crcFailureCount TotalChunkCount:$totalChunkCount -"
+    else ""
+
+    val errorMessage = if (e.message != null)
+      crcErrorLogString + e.message
+    else
+      crcErrorLogString + "Something went wrong in Verifier: $rootCause"
```

Comment:
This can be simplified like 
val errorMessage = crcErrorLogString + (e.message ?: "Something went wrong in Verifier: $rootCause")

Reply:
done